projects/ng-components-docs/src/app/examples/components/modals/modals.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="modal static" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header custom-header">
+                Title
+            </div>
+            <div class="modal-body custom-body">
+                Body
+            </div>
+            <div class="modal-footer custom-action">
+                <button type="button" class="btn btn-primary" data-dismiss="modal">
+                    Close
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
Why do you have a bootstrap modal here?

Reply:
this is a static modal that will be displayed on the page, you can notice static class on the top, this is the result
![image](https://user-images.githubusercontent.com/14928743/83129868-0560cc80-a110-11ea-9d24-f1225a59aa35.png)